openapi-cli/src/main/java/io/ballerina/openapi/cmd/OpenApiCmd.java

```diff
@@ -58,6 +59,9 @@
     @CommandLine.Option(names = {"-i", "--input"}, description = "Generating the client and service both files")
     private boolean inputPath;
 
+    @CommandLine.Option(names = {"--prefix"}, description = "Location of the file which contains any prefix to add")
+    private String prefixPath;
```

Comment:
prefixFilePath?

Reply:
Seems like we are adding licenceFilePath. Shall we use that name - WDYT?

Reply:
[Here](https://github.com/ballerina-platform/ballerina-openapi/issues/330) it is suggested to use prefix.

Reply:
Updated